docs/awsome.md

```diff
@@ -0,0 +1,154 @@
+# The Helm Guide to Writing Awesome Charts
+
+A Helm Chart provides a recipe for installing and running a containerized application inside of Kubernetes. This guide explains how to write an outstanding Chart.
+
+## Creating a New Chart
+
+You can create a new chart using the `helm create` command. This will put the chart in your workspace, which is the perfect place for trying it out. You may choose to use the `helm edit` command to editor your chart, or you may be more comfortable editing directly with your favorite editor.
+
+## A Brief Anatomy of a Chart
+
+Inside of your newly created chart, there are three pieces:
+
+- The `Chart.yaml` file contains descriptive data about a chart.
+- The `manifests/` directory contains one or more Kubernetes manifest files.
+- A `README.md` file, which is a Markdown-formatted file that explains how to configure your chart.
+
+This guide walks you through the process of creating those parts.
+
+## Naming Your Chart
```

Comment:
That's kinda why we are using the repo structure in helm so users can namespace their charts if they want to. See #118 and #162